src/components/Input/Input.tsx

```diff
@@ -20,19 +25,35 @@ interface Props extends Omit<React.InputHTMLAttributes<HTMLInputElement>, 'size'
     ref?: React.RefObject<HTMLInputElement>;
 }
 
-const { block } = bem('Input', styles);
+const { elem } = bem('Input', styles);
 
 export const Input: React.FC<Props> = React.forwardRef((props, ref) => {
-    const { children, context, disabled, isBlock, size, type, value, ...rest } = props;
+    const {
+        children,
+        context,
+        disabled,
+        isBlock,
+        size,
+        type,
+        value,
+        labelText,
+        labelClassName,
+        ...rest
+    } = props;
     return (
-        <input
-            {...rest}
-            {...block(props)}
-            ref={ref}
-            type={type}
-            disabled={disabled}
-            value={value}
-        />
+        <div>
+            {labelText && (
+                <Text {...elem('label', { elemClassName: labelClassName })}>{labelText}</Text>
```

Comment:
Tell me please why we need `labelClassName`?

Reply:
@mukiienko to have an option to change label style

Reply:
cool, I don't know that the BEM library has this option. Then I don't have any comments